Example/Tests/Tests.swift

```diff
@@ -94,7 +94,7 @@ class Tests: XCTestCase {
         .catch { error in
             XCTAssertEqual(
                 error.localizedDescription,
-                "Authorization Error: The operation couldn’t be completed. (org.openid.appauth.general error -6.)"
+                "Authorization Error: invalid_client: The client secret supplied for a confidential client is invalid."
```

Comment:
Since we are removing client_secret (#85), can this be removed?

Reply:
Lets keep this test until #85 is resolved.